### PR TITLE
release v1.26.0

### PR DESCRIPTION
**Pull request by SaitejaTamma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature

/Kind Release

> /kind hotfix

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/docs/Guides/metrics.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ If you are running a [Prometheus](https://prometheus.io/) instance you just need
 
 {{% feature publishVersion="1.26.0" %}}
 If your Prometheus metrics collection agent requires that you scrape from the pods directly(such as with [Google Cloud Managed Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus)), then the metrics ports for the controller and allocator will both be named `http` and exposed on `8080`. In the case of the allocator, the port name and number can be overriden with the `agones.allocator.serviceMetrics.http.portName` and `agones.allocator.serviceMetrics.http.port` helm chart values.
-{{% /feature %}}
```
Comment (markmandel) [inline site/content/en/docs/Guides/metrics.md:22]: Looks like the test is failing because you've removed the bottom tag for `feature`, but not the top.

Comment (SaitejaTamma) [reply site/content/en/docs/Guides/metrics.md:22]: Ya got it, i started searching with this </% /feature %>  so I missed for this one.

Comment (markmandel) [reply site/content/en/docs/Guides/metrics.md:22]: Looks like you fixed the below one, but not this one.

File: site/content/en/docs/Installation/Install Agones/helm.md [hunk 2]
```diff
@@ -212,14 +212,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                                                                                                                                                     | `false`                            |
 | `agones.controller.allocationBatchWaitTime`              | Wait time between each allocation batch when performing allocations in controller mode                                                                                                                                  | `500ms`                            |
 
-{{% feature publishVersion="1.26.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Installation/Install Agones/helm.md:215]: This shouldn't be removed. It should be bumped to 1.27.0 and the lines in this table should be merged into the larger table above.

File: site/content/en/docs/Reference/agones_crd_api_reference.html [hunk 3]
```diff
@@ -3024,8 +3024,6 @@ <h3 id="allocation.agones.dev/v1.PlayerSelector">PlayerSelector
 <p><em>
 Generated with <code>gen-crd-api-reference-docs</code>.
 </em></p>
-{{% /feature %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/agones_crd_api_reference.html:3027]: Did you modify this file by hand? It should be autogenerated.

File: site/content/en/docs/Reference/gameserverallocation.md [hunk 4]
```diff
@@ -127,8 +127,6 @@ spec:
       map:  garden22
   {{< /tab >}}
 {{< /tabpane >}}
-{{% /feature %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/gameserverallocation.md:130]: Where is the opening tag for this feature block?

File: site/layouts/partials/navbar.html [hunk 5]
```diff
@@ -27,6 +27,7 @@
 				<div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
 					<a class="dropdown-item" href="https://development.agones.dev">Development</a>
 					<a class="dropdown-item" href="https://agones.dev">{{ .Site.Params.release_version }}</a>
+					<a class="dropdown-item" href="https://1-24-0.agones.dev">1.25.0</a>
```
Comment (roberthbailey) [inline site/layouts/partials/navbar.html:30]: Should be 1-25 not 1-24

File: site/content/en/docs/Reference/gameserverallocation.md [hunk 6]
```diff
@@ -11,7 +11,6 @@ A full `GameServerAllocation` specification is available below and in the
 {{< ghlink href="/examples/gameserverallocation.yaml" >}}example folder{{< /ghlink >}} for reference:
 
 
-{{% feature expiryVersion="1.26.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/gameserverallocation.md:14]: You need to delete everything between the tags for things that are expiring.

**Review by roberthbailey — APPROVED** [review]

**Comment by roberthbailey** [comment]
The test that is failing is `test-gen-api-docs`. I think it's because you manually modified `agones_crd_api_reference.html`. Did you run `make gen-api-docs`?

**Comment by SaitejaTamma** [comment]
> The test that is failing is `test-gen-api-docs`. I think it's because you manually modified `agones_crd_api_reference.html`. Did you run `make gen-api-docs`?
I removed it manually and added back, shall i remove and run the `make gen-api-docs` again

**Comment by roberthbailey** [comment]
I think you can just run the make command again which will overwrite it. You can also run `make test-gen-api-docs` locally to make sure that it's fixed before pushing your changes to github.

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**